### DISCUSSION TITLE
Just enabling fork or warp when the instructions or notes include a link

### DISCUSSION THREAD

**Discussion by NoobTracker** [ScratchAddons/ScratchAddons#191] [Discussion]
So my idea is that when the description contains either the string `forkphorus.github.io` or `turbowarp.org` or `turbowarp.github.io`, it will just replace the stage without asking (in case it's enabled and it is by default). There would be a button to view it on Scratch anyway by adding `?ignoreplayerlinks=true` to the URL .... I think that's it. Oh, I forgot, there would be the option to prefer TurboWarp over forkphorus, turned off by default. 

So now my question is if that makes sense. I'm not sure if there's an edge case where it makes no sense.

**Note**: I'm not gonna add this currently, first I just want that the current thing is merged.

**Comment by Hans5958** [comment]
I don't think we can do this. There is a chance that the project author just link it to make the users aware that there is an alternative, not to force an alternative.

**Comment by NoobTracker** [comment]
Well that chance is pretty low and in that case you can just click "Don't care about that stuff and just use Scratch".

**Comment by NoobTracker** [comment]
And, as I said, everything is optional. I may even add stuff that will only search for recommended players if the project doesn't use cloud variables.

**Comment by NoobTracker** [comment]
@Hans5958 Currently I'm just trying to do it and it works pretty well so far, it just checks the description.

**Comment by NoobTracker** [comment]
Is there a better way of adding the `?ignoreplayerlinks=true` to the URL then just adding that string to the location?

**Comment by Hans5958** [comment]
Sure, sure. Also, I just realised that this won't be activated by default so I guess it is fine.

About your latest comment, try this.
https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/append

Also, you put the wrong title and label. cc: @apple502j or whoever that could fix the labels

**Comment by NoobTracker** [comment]
And I think that when someone recommends fork, that usually makes sense. And even if it was enabled by default, the whole addon isn't.
I'll look at that, thank you for the link!

**Comment by NoobTracker** [comment]
Label for what?

**Comment by NoobTracker** [comment]
@Hans5958 Okay, that link helped me a lot! I didn't know how the ? is called so I didn't know what to search for.

**Comment by Hans5958** [comment]
> Label for what?

The label. It should be new-addons and the title should be starting with [New addons].

> Okay, that link helped me a lot! I didn't know how the ? is called so I didn't know what to search for. 

Those all call URL parameters, fyi.

**Comment by NoobTracker** [comment]
Ah okay but it isn't a new addon and I wanted to discuss if that makes sense so I dunno.

**Comment by Hans5958** [comment]
I see. I think ideas, even if it's make sense or not, are dropped as new-addons, but IDK.

**Comment by NoobTracker** [comment]
Maybe there should be a label for improvement ideas for addons.
